h/accounts/test/views_test.py

```diff
@@ -68,34 +73,36 @@ def test_ajax_form_handles_http_error_as_error():
 
 def test_ajax_form_sets_failure_status_on_errors():
     request = DummyRequest()
-    result = ajax_form(request, {'errors': 'data'})
+    result = ajax_form(request, {'errors': {}})
```

Comment:
These changes are just to make it clear that errors is supposed to be a dict (and only a dict) now.